File: CodingTestPlatform/BOJ/1_Bronze/1/9093/9093.py
# 단어 뒤집기
# https://www.acmicpc.net/problem/9093

import sys
input = sys.stdin.readline

cnt = int(input().rstrip())
for i in range(cnt):
    s = input().rstrip().split()
    rst = []
    for j in s:
        rst.append(j[::-1])
    print(' '.join(rst))

File: CodingTestPlatform/BOJ/1_Bronze/1/9093/9093.js
/**
 * 단어 뒤집기
 * Bronze1
 * https://www.acmicpc.net/problem/9093
 */

const [T, ...input] = require('fs').readFileSync('/dev/stdin').toString().trim().split("\n");

const result = input.map((str) => {
    return str.split(" ").map((str2) => {
        return str2.split("").reverse().join("");
    }).join(" ");
}).join("\n");

console.log(result);